naga/tests/snapshots.rs

```diff
@@ -39,61 +39,60 @@ impl Default for SpvOutVersion {
 }
 
 #[derive(Default, serde::Deserialize)]
+#[serde(default)]
 struct SpirvOutParameters {
     version: SpvOutVersion,
-    #[serde(default)]
     capabilities: naga::FastHashSet<spirv::Capability>,
-    #[serde(default)]
     debug: bool,
-    #[serde(default)]
     adjust_coordinate_space: bool,
-    #[serde(default)]
     force_point_size: bool,
-    #[serde(default)]
     clamp_frag_depth: bool,
-    #[serde(default)]
     separate_entry_points: bool,
-    #[serde(default)]
     #[cfg(all(feature = "deserialize", spv_out))]
     binding_map: naga::back::spv::BindingMap,
 }
 
 #[derive(Default, serde::Deserialize)]
+#[serde(default)]
 struct WgslOutParameters {
-    #[serde(default)]
     explicit_types: bool,
 }
 
 #[derive(Default, serde::Deserialize)]
+#[serde(default)]
 struct Parameters {
-    #[serde(default)]
+    // -- GOD MODE --
     god_mode: bool,
```

Comment:
I know this is entirely peripheral to this PR, but this is not at all self-describing.

Reply:
I wasn't trying to describe it, just put some section titles